requirements/extra.in

```diff
@@ -1,7 +1,5 @@
 confluent-kafka
-colorama
+ess-streaming-data-types
 scipp
 scippneutron
 plopp
-h5py
-ess-streaming-data-types
```

Comment:
Given that only ESS will use Beamlime, is there any point in making these optional/extra? Might be simpler to just put these into `pyproject.toml`, at least until someone actually wants to use this for something else (which is really unlikely).

Reply:
I thought it should be done as they become actual dependencies of the package,
but and then I thought it doesn't really matter... so I added them...! Thanks!